report-jtreg-lib/src/main/java/io/jenkins/plugins/report/jtreg/Constants.java

```diff
@@ -29,6 +29,6 @@ final public class Constants {
     public static final String REPORT_TESTS_LIST_JSON = "tests-list.json";
     public static final String IRRELEVANT_GLOB_STRING = "report-{runtime,devtools,compiler}.xml.gz";
     public static final double VAGUE_QUERY_THRESHOLD = 0.5;
-
     public static final int VAGUE_QUERY_LENGTH_THRESHOLD = 4;
+    public static final String ESCAPE_DIRECTORY_REGEX = "((\\.\\./)+.*(\\.\\./)*.*)|((\\.\\./)*.*(\\.\\./)+.*)";
```

Comment:
I wil merge this as it is. But the rework of shoudl be needed.
Again,. from my own experiments, checking  input like this may be missleading.

eg correctdir/correctdir/../fileInFirstCorrectDir is pretty correct 
Eg I'm checking the bash injection in the CGI wrapper by checking for "|" and thus literally killing this tool. Btrw, any ideas how to fix that "|"  (https://github.com/jenkinsci/report-jtreg-plugin/blob/master/report-jtreg-service/src/main/java/io/jenkins/plugins/report/jtreg/main/web/ContextExecutingHandler.java#L114)